src/test/java/org/gridsuite/shortcircuit/server/ShortCircuitAnalysisControllerTest.java

```diff
@@ -340,6 +354,15 @@ void runTest() throws Exception {
             shortCircuitAnalysisMockedStatic.when(ShortCircuitAnalysis::find).thenReturn(runner);
             when(runner.getName()).thenReturn("providerTest");
 
+            // mock s3 client for run with debug
+            doReturn(PutObjectResponse.builder().build()).when(s3Client).putObject(eq(PutObjectRequest.builder().build()), any(RequestBody.class));
```

Comment:
Implement a specific test for debug. This one is already huge and there's a sonar warning

Reply:
DONE